src/components/ItaliaTheme/Blocks/Common/SearchFilters/DateFilter.jsx

```diff
@@ -251,6 +273,8 @@ const DateFilter = (props) => {
         hideKeyboardShortcutsPanel={true}
         showClearDates
         phrases={getDateRangePickerPhrases(intl)}
+        customArrowIcon={customArrowIcon()}
+        customCloseIcon={customCloseIcon()}
```

Comment:
Perché non mettere direttamente le icone qui dentro?

Tipo così

```suggestion
        customArrowIcon={
          <Icon
            icon="it-arrow-right"
            color="white"
            title={intl.formatMessage(messages.roleDescription)}
          />
        }
```

O in alternativa se diventa troppo brutto farei almeno in modo che `customArrowIcon` non sia una funzione ma una semplice variabile, tanto non ha realmente dei parametri, cioè non hai bisogno di generarla con una funzione per renderla mutabile in base a certi parametri.